sdk/cosmos/azure-cosmos-kafka-connect/pom.xml

```diff
@@ -189,7 +189,7 @@ Licensed under the MIT License.
               <includes>
                 <include>com.azure:*</include>
                 <include>org.apache.kafka:connect-api:[3.6.0]</include> <!-- {x-include-update;cosmos_org.apache.kafka:connect-api;external_dependency} -->
-                <include>cosmos_io.confluent:kafka-connect-maven-plugin:[0.12.0]</include> <!-- {x-include-update;cosmos_io.confluent:kafka-connect-maven-plugin;external_dependency} -->
+                <include>io.confluent:kafka-connect-maven-plugin:[0.12.0]</include> <!-- {x-include-update;cosmos_io.confluent:kafka-connect-maven-plugin;external_dependency} -->
```

Comment:
The tag on the right still says cosmos_io, is that correct ?

Reply:
Yeah, this was actually incorrect before.